src/API/Site/Controllers/Ads/IncentiveCreditsController.php

```diff
@@ -0,0 +1,124 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\Ads;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Middleware;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\ContainerAwareTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\Interfaces\ContainerAwareInterface;
+use Exception;
+use WP_REST_Request as Request;
+use WP_REST_Response as Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class IncentiveCreditsController
+ *
+ * Handles fetching Google Ads incentive credits.
+ *
+ * ContainerAware used for:
+ * - Middleware
+ *
+ * @since x.x.x
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\Ads
+ */
+class IncentiveCreditsController extends BaseController implements ContainerAwareInterface {
+
+	use ContainerAwareTrait;
+
+	/**
+	 * Register rest routes with WordPress.
+	 */
+	public function register_routes(): void {
+		$this->register_route(
+			'ads/incentive-credits',
+			[
+				[
+					'methods'             => 'GET',
```

Comment:
`'GET'` should be able to be replaced with `TransportMethods::READABLE` for better consistency.

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/2836/commits/4c4c80343229f93b61ece6b31483e7f55a09b6d5.